scripts/maintenance/checkUniversalVerifierSingleNetwork.ts

```diff
@@ -0,0 +1,65 @@
+import { getStateContractAddress, Logger } from "../../helpers/helperUtils";
+import { contractsInfo } from "../../helpers/constants";
+import hre, { ethers } from "hardhat";
+import {
+  setZKPRequest_KYCAgeCredential,
+  submitZKPResponses_KYCAgeCredential,
+} from "../upgrade/verifiers/helpers/testVerifier";
+import { Contract } from "ethers";
+
+const universalVerifierAddress = "<put-your-universal-verifier-address>";
```

Comment:
Please load these addresses from constants.

Reply:
This script is used to load old verifiers and test them for backward compatibility. These addresses can be other than unified addresses. But I'll put unified addresses by default.